### PR TITLE
add example data, edit config

### PR DESCRIPTION
**Pull request by hgloeckner**
Added some example data and changed the path in the standard config such that people that clone the repo already get some dropsonde data to play with (also for us to be able to do tests later). The data I put is fairly arbitrary, so feel free to put different data

### CODE REVIEW

File: example_data/HALO/20200119/.1_Aircraft_20200119_155829.txt.swp
Comment (d70-t) [inline example_data/HALO/20200119/.1_Aircraft_20200119_155829.txt.swp:1]: This swp file maybe was committed by accident...

File: example_data/HALO/20200119/Level_2/HALO-(AC)3_HALO_2020-01-19T16:55:14.000000_20200119_190140094_Level_2.nc
Comment (Geet-George) [inline example_data/HALO/20200119/Level_2/HALO-(AC)3_HALO_2020-01-19T16:55:14.000000_20200119_190140094_Level_2.nc:1]: This file needn't be there as part of sample data.

File: example_data/HALO/20200122/Level_2/HALO-(AC)3_HALO_2020-01-22T15:18:15.000000_20200122_192510289_Level_2.nc
Comment (Geet-George) [inline example_data/HALO/20200122/Level_2/HALO-(AC)3_HALO_2020-01-22T15:18:15.000000_20200122_192510289_Level_2.nc:1]: This too. I think all Level-2 directories can be removed.

Comment (hgloeckner) [reply example_data/HALO/20200122/Level_2/HALO-(AC)3_HALO_2020-01-22T15:18:15.000000_20200122_192510289_Level_2.nc:1]: Yes - I removed them.

File: example_data/HALO/20200119/.1_Aircraft_20200119_155829.txt.swp
Comment (hgloeckner) [reply example_data/HALO/20200119/.1_Aircraft_20200119_155829.txt.swp:1]: You are right, thanks for pointing it out :)

**Comment by Geet-George** [comment]
closes #123 too.

**Pull request merged**